### PR TITLE
Add default entity info as include for kickstart

### PR DESCRIPTION
**Pull request by mooreds**
This is a fix for https://github.com/FusionAuth/fusionauth-issues/issues/459

Also, I included the default entity ids rather than pointing a link.

Reviewed other defaults (grepping for static constants in the java-client, looking at Kickstart java).

### CODE REVIEW

File: site/docs/v1/tech/shared/_default-entities.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+* The Default Tenant
```
Comment (jobannon) [inline site/docs/v1/tech/shared/_default-entities.adoc:1]: should tenant be caps?

File: site/docs/v1/tech/shared/_default-entities.adoc [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+* The FusionAuth application, which resides in the Default tenant. This will always have the Id `3c219e58-ed0e-4b18-ad48-f4f92793ae32`.
+* The Default theme, which is read only. This will always have the Id `75a068fd-e94b-451a-9aeb-3ddb9a3b5987`.
+* The FusionAuth application roles, which may be assigned to users, but are immutable. View the link:/docs/v1/tech/core-concepts/roles/[roles documentation] for the fixed Ids of each role.
+* Three OpenID Connect shadow keys. These may not be modified or removed. When any of these keys are selected as the signing key for the Id Token the client secret defined by the FusionAuth Application will be used to sign the token.
```
Comment (jobannon) [inline site/docs/v1/tech/shared/_default-entities.adoc:5]: This sentence is confusing to me and might be related to my lack of context.

File: site/docs/v1/tech/installation-guide/kickstart.adoc [hunk 3]
```diff
@@ -279,9 +279,13 @@ In this example we will modify the restricted API key example from above to furt
 
 === Referencing Default Entities
 
-Review the link:https://fusionauth.io/docs/v1/tech/reference/limitations/#default-entities[default entities] documentation for entities with fixed Ids such as the default theme or FusionAuth application. 
+There are some entities in FusionAuth with fixed Ids, such as the default theme or FusionAuth application.
```
Comment (jobannon) [inline site/docs/v1/tech/installation-guide/kickstart.adoc:282]: Using the word `entity` can be confusing as it shares the same namespace as `entities`, `entity types`, and `entity management`

**Review by jobannon — APPROVED** [review]
LGTM! Some minor thoughts....

File: site/docs/v1/tech/installation-guide/kickstart.adoc (hunk 3 above)
Comment (mooreds) [reply site/docs/v1/tech/installation-guide/kickstart.adoc:282]: fair

**Pull request merged**